rasa/core/processor.py

```diff
@@ -389,7 +400,7 @@ def _should_handle_message(tracker: DialogueStateTracker):
         )
 
     async def _predict_and_execute_next_action(
-        self, message: UserMessage, tracker: DialogueStateTracker
+        self, output_channel: Optional[OutputChannel], tracker: DialogueStateTracker
```

Comment:
is it really `Optional`?
```suggestion
        self, output_channel: OutputChannel, tracker: DialogueStateTracker
```